### PR TITLE
manual test for PaymentResponse.shippingAddress attr

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: payment-request/payment-response/shippingAddress-attribute-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+async function runManualTest(button, options = {}, expected = {}, id) {
+  button.disabled = true;
+  const { request, response } = await getPaymentRequestResponse(options, id);
+  await response.complete();
+  test(() => {
+    assert_equals(response.requestId, request.id, `Expected ids to match`);
+    const { shippingAddress: addr } = request;
+    assert_true(
+      addr.addressLine instanceof PaymentAddress,
```
Comment (edenchuang) [inline payment-request/payment-response/shippingAddress-attribute-manual.https.html]: I guess it should be "addr instanceof PaymentAddress", not addr.addressLine.

File: payment-request/payment-response/shippingAddress-attribute-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      </button>
+  </li>
+  <li>
+    <button onclick="checkNullShippingAddress(this, {requestShipping: false})">
+      If the requestShipping member is false, then shippingAddress must be null.
+    </button>
+  </li>
+  <li>
+    <button onclick="runManualTest(this, {requestShipping: true}, { shippingAddress: 'pass' }).then(done)">
+      If the requestShipping member is true, then shippingAddress will be the full and final shipping address is "wpt street" in "Kabul, Afghanistan", zip/postal code 1001.
```
Comment (edenchuang) [inline payment-request/payment-response/shippingAddress-attribute-manual.https.html]: s/wpt street/1 wpt street

**Review by edenchuang — CHANGES_REQUESTED** [review]

File: payment-request/payment-response/shippingAddress-attribute-manual.https.html (hunk 1 above)
Comment (marcoscaceres) [inline payment-request/payment-response/shippingAddress-attribute-manual.https.html]: Good catch!

File: payment-request/payment-response/shippingAddress-attribute-manual.https.html (hunk 2 above)
Comment (marcoscaceres) [inline payment-request/payment-response/shippingAddress-attribute-manual.https.html]: fixing

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 07:27:27
Finished: 2017-08-21 07:33:33



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17599)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266697790)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 07:27:27
Finished: 2017-08-21 07:33:33



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17599)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266697790)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 07:27:27
Finished: 2017-08-21 07:33:33



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17599)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266697790)

**Review by edenchuang — APPROVED** [review]
LGTM

**Pull request merged**